end.sh

```diff
@@ -23,4 +23,9 @@ echo "Killing $NAME slurm job on ${RESOURCE}"
 ssh ${RESOURCE} "squeue --name=$NAME --user=$FORWARD_USERNAME -o '%A' -h | xargs --no-run-if-empty /usr/bin/scancel"
 
 echo "Killing listeners on ${RESOURCE}"
-ssh ${RESOURCE} "/usr/sbin/lsof -i :$PORT -t | xargs --no-run-if-empty kill"
+if [[ "${RESOURCE}" == "sherlock" ]]
```

Comment:
Could we instead make this a Boolean variable that is not resource specific? It can go into the config file.

Reply:
fixed with boolean operator 
6aa907f